test/e2e/audit/audit_log_test.go

```diff
@@ -36,25 +37,28 @@ func TestAuditLogs(t *testing.T) {
 	t.Parallel()
 	framework.Suite(t, "control-plane")
 
-	server := framework.PrivateKcpServer(t, framework.WithCustomArguments([]string{"--audit-log-path", "./audit-log", "--audit-policy-file", "./policy.yaml"}...))
-
-	ctx, cancelFunc := context.WithCancel(context.Background())
+	artifactDir, dataDir, err := framework.ScratchDirs(t)
+	require.NoError(t, err)
 
-	t.Cleanup(func() {
```

Comment:
I'm assuming we lost this since we defer cleanup to the scratch-dir cleanup routines?

Reply:
Yes since the audit-log ends up in the regular artifactDir which is [created via testing.TempDir](https://github.com/kcp-dev/kcp/blob/main/test/e2e/framework/util.go#L114) it'll be cleaned with the other test output unless ARTIFACT_DIR is specified.